users/README.md

```diff
@@ -43,6 +43,7 @@ number and email address.
 |   [GET /users/:userId/badges](#get-usersidbadges)   | Returns badges assigned to the user  |
 |             [POST /users](#post-users)              |          Creates a new User          |
 |        [PATCH /users/self](#patch-usersself)        |       Updates data of the User       |
+|        [PATCH /users/:id/roles](#patch-usersidroles)        |       Updates user roles     |
```

Comment:
the formatting seems to be out of place 🤔

Reply:
Done can you please check in latest commit